projects/logging/examples/logging.yml

```diff
@@ -2,7 +2,7 @@ kernel:
   image: "linuxkit/kernel:4.9.x"
   cmdline: "console=ttyS0 console=tty0 page_poison=1"
 init:
-  - linuxkit/init:1b8a7e394d2ec2f1fdb4d67645829d1b5bdca037 # with runc, logwrite, startmemlogd
+  - linuxkit/init:b5c88b78cd9cc73ed83b45f66bc9de618223768a # NOTE: custom init with logwrite ctr/runc, startmemlogd
```

Comment:
@riyazdf thanks! I sent you my hub ID on slack. Maybe we could create `linuxkitprojects/logging-memlogd` as well? It's currently in `linuxkit/memlogd`.